sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -9,6 +9,65 @@ import { AbortSignalLike } from '@azure/abort-controller';
 // @public
 export type CancelOnProgress = () => void;
 
+// @public
+export function createGetLroStatusFromResponse<TResult>(lroPrimitives: LongRunningOperation<TResult>, config: LroConfig, lroResourceLocationConfig?: LroResourceLocationConfig): GetLroStatusFromResponse<TResult>;
+
+// @public
+export type GetLroStatusFromResponse<T> = (rawResponse: RawResponse, flatResponse: T) => LroStatus<T>;
```

Comment:
Can we use more specific `T` types? like how we were doing with `TResult`, for example?

Reply:
no longer part of the public API